package.json

```diff
@@ -170,7 +170,7 @@
     "commander": "^2.20.0",
     "concurrently": "^4.1.1",
     "cross-env": "^5.2.0",
-    "csstype": "^3.0.11",
+    "csstype": "3.0.11",
```

Comment:
Simon added a DangerJS check to verify `dependencies` are pinned. It only runs against `package.json` files with changes in commit history, so old `package.json` files that aren't pinned are ignored. I use lodash in my package.json, so I had to pin it. When pinning lodash, yarn complains that all lodashes need to be pinned. In doing so, I touch a package.json file with csstype so that starts to complain. I pin csstype and we're good to go.

Reply:
For extra context, even though we ignore the root package.json file in the dangerJS check, when we change it at the workspace level, manypkg check makes it required that the monorepo uses the same version across all packages and root. Hence you have to change it here too.

Reply:
Ohhhh ok that makes sense! Thanks for explaining!